packages/@aws-cdk/aws-stepfunctions/lib/state-machine.ts

```diff
@@ -500,6 +485,27 @@ export class StateMachine extends StateMachineBase {
       enabled: true,
     };
   }
+
+  private addDistributedMapStatePolicies(stateMachineName?: string) {
```

Comment:
Making this optional and using the wildcard `*` later on seems like it's not required?
This is a private method and we only ever call it with the argument.